src/App.js

```diff
@@ -1,19 +1,44 @@
 import React from 'react';
 import './App.css';
 
+import todos from './api/todos';
 import users from './api/users';
+import { TodoList } from './components/TodoList/TodoList';
+import { AddTodoForm } from './components/AddTodo/AddTodoForm';
 
-function App() {
-  return (
-    <div className="App">
-      <h1>Add todo form</h1>
+class App extends React.Component {
+  state = {
+    preparedTodos: todos.map(todo => ({
+      ...todo,
+      user: users.find(({ id }) => id === todo.userId),
+    })),
+    authors: users,
```

Comment:
no need to save users in state if you don't change them
just import them, it's enough